### DISCUSSION TITLE
Task option, Run when players has x hours online

### DISCUSSION THREAD

**Discussion by Zoro189** [gameserverapp/Platform#168] [Suggestions]
Think it could be useful trigger option to be able to execute command after x hours player played on specific server and maybe also option for whole cluster. And shouldn't be to much work since GSA is already tracking time spend on server

**Comment by mvaessen** [comment]
A feature like this is in fact a lot of work.
Mainly because the logic for getting this working properly is very complex.
Using "simple" logic, where it will execute when someone played over 100 hours (`$time > 100`), will result in infinite executes. Since $time is always more than 100 after a certain point. So the system needs to keep track of achieving certain milestones. Tasks aren't meant for that.

I am planning to add an achievement system, that is more suitable to trigger actions based on achieving certain milestones.
There is no ETA when Achievements will be available, but I'm hoping to release it somewhere this year.
